compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -1475,9 +1475,6 @@ class OMR_EXTENSIBLE CodeGenerator
    bool getSupportsArraySet() {return _flags1.testAny(SupportsArraySet);}
    void setSupportsArraySet() {_flags1.set(SupportsArraySet);}
 
-   bool getSupportsArraySetToZero() {return _flags3.testAny(SupportsArraySetToZero);}
```

Comment:
I prefer not to; as this PR is only to enable arrayset on X86.
With this PR, X declares supports for both arrayset to zero and arbitrary arrayset. Z already declares supports for both. ARM supports neither. The only platform violates this proposed rule is P, which, under TR_AggressiveOpts, only declares support for arbitrary arrayset. I'll contact folks in charge of P and see if they would add declaration for supports arrayset to zero.